lang/main.json

```diff
@@ -1225,6 +1225,8 @@
         "labelToolTip": "The meeting is being transcribed",
         "off": "Transcribing stopped",
         "pending": "Preparing to transcribe the meeting...",
+        "sourceLanguageDesc": "Currently the meeting language is set to <b>{{sourceLanguage}}</b>. <br/> You can change it from {{here}}",
```

Comment:
Seems that `here` placeholder is not defined when  this translation is used.

Reply:
removed